src/main/resources/application-dev.yml

```diff
@@ -7,3 +7,22 @@ spring:
   flyway:
     enabled: true
     locations: classpath:/db/migration
+
+APP_DB_URL: "jdbc:postgresql://127.0.0.1:5433/jpc?reWriteBatchedInserts=true&user=jpc&password=letmein"
```

Comment:
Brought these in here so the profile can be specified when running the integration tests.

This profile also adds some test data via Flywaydb migrations.